src/style.css

```diff
@@ -1,3 +1,58 @@
+/* stylelint-disable declaration-block-no-redundant-longhand-properties */
 body {
+  font-family: Roboto, sans-serif;
```

Comment:
Consider adding a fallback font-family for the case where 'Roboto' is not available. For example, 'font-family: Roboto, Arial, sans-serif;'. This ensures that your text is still readable even if the primary font fails to load.